inference-engine/src/vpu/graph_transformer/src/stages/loop_end.cpp

```diff
@@ -5,17 +5,13 @@
 #include "vpu/frontend/frontend.hpp"
 #include "vpu/stages/iteration_rule.hpp"
 
-#include <utility>
```

Comment:
are you going to change copyright years?

Reply:
Yes

Reply:
Fixed